Add extra guidance on mentored pair programming (#1133)
CodeYourFuture/curriculum: org-cyf-guides/content/pair-programming/_index.md

+++
title = 'Pair programming'
description = 'Practice writing code and getting support'
emoji = '👥'
+++

Pair programming is an excellent way to develop programming and communication skills.

> It is often much easier to work through something when working on something 1-to-1.

Reviewer: Minor: Maybe this could be something like "It is often much easier to work through a problem when working with someone 1-to-1." ?

> It is often much easier to work through a problem when working with someone 1-to-1.

Reviewer 2: ```suggestion
It can be easier to work through a problem when working with someone 1-to-1.
```

> It also helps our trainees to prepare for technical interviews when they'll need to code in front of other people.

Reviewer 2: ```suggestion
Pair programming prepares our trainees for technical interviews when they'll 
 code in front of other people.
```

> It is also helps our trainees to prepare for technical interviews when they'll need to code in front of other people.

Reviewer: It ~is~ also
